src/main/java/seedu/address/logic/commands/AddTimetableCommand.java

```diff
@@ -0,0 +1,18 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import seedu.address.logic.CommandHistory;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+public class AddTimetableCommand extends Command {
+  public static final String COMMAND_WORD = "add_timetable";
+  public static final String MESSAGE_SUCCESS = "New timetable added: %1$s";
+  public static final String MESSAGE_USAGE = COMMAND_WORD;
+
```

Comment:
Maybe you need to consider the exception of the user already having a timetable in the system: Should the system replace the timetable? Or should the system throw an error?